lib/main.coffee

```diff
@@ -28,11 +33,37 @@ module.exports =
       lint: (textEditor) =>
         filePath = textEditor.getPath()
         fileText = textEditor.getText()
+        fileDir = _path2.default.dirname(filePath)
+
+        configFile = helpers.findCached fileDir, [
```

Comment:
Basically, here i am just looking for a config file anywhere up the folder structure from the current file to the project root, for a tidy cfg file.
Using absolute paths would make it so the config was not project specific (I think) ?

Have also changed this so there is only one cfg filename option(.tidycfg.txt) by default, but the user can set the name in config to whatever suits them best.